src/ripple/nodestore/impl/codec.h

```diff
@@ -235,6 +235,80 @@ nodeobject_decompress (void const* in,
         write(os, is(512), 512);
         break;
     }
+    case 5: // compressed v2 inner node
```

Comment:
There's a numbering system for node types that we're being consistent with here.  4 is a leaf node representing a transaction with metadata.  This can be found over in `SHAMapAbstractNode::make`.